docs/installation.md

```diff
@@ -35,6 +35,8 @@ curl -o environment.yml https://raw.githubusercontent.com/phenology/springtime/m
 mamba env update -n springtime -f environment.yml
 ```
 
+Note that some of the R dependencies are not available as pre-compiled binaries for some operating systems and architectures. If the command above fails, follow [the step below](#install-r-dependencies) to build the R dependencies on your system.
+
```

Comment:
Note that the conda environment is also not complete. Even if you use this, you still need to run the R install instructions from below. However, without pre-installing most (indirect) dependencies, this takes ages.

Reply:
Yes, I wanted to stress the fact that even if this fails (like on OS X with Intel processor, for which r-BayesialTools is not available), you could still go ahead and compile the binaries in the following step.  But I have tried to make this more explicit, do you think it is clearer now?

Reply:
Yes! I agree the clarification was needed/helpful. My only concern was that the previous formulation seemed to suggest that the next steps weren't necessary if this step worked. The new text is much better, thanks